src/orquestra/integrations/cirq/simulator/_base.py

```diff
@@ -55,6 +55,7 @@ def __init__(
         noise_model: cirq.NOISE_MODEL_LIKE = None,
         param_resolver: cirq.ParamResolverOrSimilarType = None,
         qubit_order=cirq.ops.QubitOrder.DEFAULT,
+        normalize: bool = False,
```

Comment:
I'd rename it to `normalize_wavefunction` and include information about why it's needed in the docstring :)

Reply:
Done :-D